src/ontology/mondo-edit.obo

```diff
@@ -524968,19 +524968,18 @@ is_a: MONDO:0015137 {source="https://orcid.org/0000-0001-5208-3432"} ! periodic
 
 [Term]
 id: MONDO:0957404
-name: pyogenic autoinflammatory syndrome of childhood
-comment: Reason of obsoletion: out of scope - MONDO:excludeHistoricalDisease. Term to consider: -none. This term's classification was reviewed in the context of the Strategic Refinement project (2023) and was determined to be excluded from the 'connective tissue disorder' (MONDO:0018153) ontology branch (https://orcid.org/0000-0002-1780-5230)
+name: obsolete pyogenic autoinflammatory syndrome of childhood
+comment: This term's classification was reviewed in the context of the Strategic Refinement project (2023) and was determined to be excluded from the 'connective tissue disorder' (MONDO:0018153) ontology branch (https://orcid.org/0000-0002-1780-5230)
 subset: gard_rare {source="GARD:21459"}
-subset: obsoletion_candidate
 subset: orphanet_rare {source="Orphanet:324942"}
 subset: rare
 xref: GARD:21459 {source="Orphanet:324942"}
-xref: Orphanet:324942 {source="MONDO:equivalentTo"}
-is_a: MONDO:0019751 {source="https://orcid.org/0000-0001-7941-2961", source="https://orcid.org/0000-0002-5002-8648"} ! autoinflammatory syndrome
+xref: Orphanet:324942 {source="MONDO:obsoleteEquivalent"}
+property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeHistoricalDisease"}
```

Comment:
Is this the same reason as used in the strategic refinement work? I think is should be: `property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeGrouping"}`

Reply:
this was not a grouping class, so it is not appropriate to add that exclusion reason.

Reply:
@twhetzel could you please review again and approve?

Reply:
Oh, even though the Orphanet page says it is a grouping term: https://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=EN&Expert=324942?

Reply:
got it! I'll add the exclusion reason

Reply:
ready for review